ptn/buttonrolebot/ui_elements/EmbedCreator.py

```diff
@@ -153,7 +153,12 @@ async def on_submit(self, interaction: discord.Interaction):
         self.embed_data.embed_author_avatar = self.author_avatar.value if self.author_avatar.value else None
 
         # validate URLs
-        if self.embed_data.embed_thumbnail is not None and not validators.url(self.embed_data.embed_thumbnail):
+        def is_valid_extension(url):
```

Comment:
Wondering if this should be in Helpers so it's accessible to future functions, but is OK here for now